libbeat/scripts/Makefile

```diff
@@ -87,8 +87,9 @@ SYSTEM_TESTS?=false ## @testing if true, "make test" and "make testsuite" run un
 STRESS_TESTS?=false ## @testing if true, "make test" and "make testsuite" run also run the stress tests
 STRESS_TEST_OPTIONS?=-timeout=20m -race -v
 GOX_OS?=linux darwin windows freebsd netbsd openbsd ## @Building List of all OS to be supported by "make crosscompile".
-GOX_OSARCH?=!darwin/arm !darwin/arm64 ## @building Space separated list of GOOS/GOARCH pairs to build by "make crosscompile".
+GOX_OSARCH?=!darwin/arm !darwin/386 !linux/386 !windows/386 !freebsd/386 !netbsd/386 !openbsd/386 !linux/ppc64 ## @building Space separated list of GOOS/GOARCH pairs to build by "make crosscompile".
 GOX_FLAGS?= ## @building Additional flags to append to the gox command used by "make crosscompile".
+GOTEST_OUTPUT_OPTIONS?=
```

Comment:
What's the reason for this new flag? A comment that explains what it does can help in the future.

Reply:
Because in the original pipeline stage are no artifacts at all. I use this option to see the result of tests as the artifacts in the build (define artifact path and then upload that to BK)

Reply:
Can you add a comment explaining that as similarly done in L91 or others?

Reply:
+